cmd/soroban-cli/src/commands/tx/fetch/fee.rs

```diff
@@ -0,0 +1,215 @@
+use super::args;
+use crate::{
+    commands::global,
+    config::network,
+    rpc,
+    xdr::{self, Hash, SorobanTransactionMetaExt, TransactionMeta},
+};
+use clap::{command, Parser};
+use prettytable::{
+    format::{FormatBuilder, LinePosition, LineSeparator},
+    Cell, Row, Table,
+};
+use serde::{Deserialize, Serialize};
+use soroban_rpc::GetTransactionResponse;
+
+#[derive(Parser, Debug, Clone)]
+#[group(skip)]
+pub struct Cmd {
+    #[command(flatten)]
+    pub(crate) args: FeeArgs,
+}
+
+#[derive(Debug, Clone, clap::Args)]
+pub struct FeeArgs {
```

Comment:
What's the benefit of setting the args up this way where they are defined separately then flattened?

Reply:
Good question! I think I had a refactor in mind to re-use the `fetch_transaction` fn from args::Args, but forgot to finish it. 

I'll get something pushed up to clean this up shortly.